scripts/start.sh

```diff
@@ -292,7 +292,15 @@ fi
 if [ "${BACKUP_ENABLED}" = true ]; then
     echo "BACKUP_ENABLED=${BACKUP_ENABLED}"
     
-    echo "$BACKUP_CRON_EXPRESSION bash /usr/local/bin/backup" > "/home/steam/server/crontab"
+    echo "$BACKUP_CRON_EXPRESSION bash /usr/local/bin/backup" >> "/home/steam/server/crontab"
```

Comment:
Need `rm -f  "/home/steam/server/crontab"` before this line and outside the if statement.
Without it everytime the container restarts it adds to the file
```
0 0 * * * bash /usr/local/bin/backup
* * * * * bash /usr/local/bin/update
0 0 * * * bash /usr/local/bin/backup
* * * * * bash /usr/local/bin/update
```

Reply:
It should work now as expected. Deleting the crontab before Backup and Update If statement